docker/chemml/src/run.py

```diff
@@ -25,25 +38,13 @@ def run_calculation(geometry_file, output_file):
     cls.load(summary=True)
     pred = cls.predict(smiles)
 
-    calculated_properties = {
-        "refractiveIndex": {
-            "label": "LL refractive index",
-            "value": pred[0],
-            "units": ""
-        },
-        "polarizability": {
-            "label": "Polarizability",
-            "value": pred[1],
-            "units": "Bohr^3"
-        },
-        "density": {
-            "label": "Density",
-            "value": pred[2],
-            "units": "Kg/m^3"
-        }
+    properties = {
+        "refractiveIndex": pred[0],
+        "polarizability": pred[1],
+        "density": pred[2]
     }
 
-    cjson['calculatedProperties'] = calculated_properties
+    cjson['properties'] = properties
```

Comment:
Should we do an `update` in case we have props that we shouldn't override?

Reply:
I thought we are reserving `properties` on the cjson to store calculation properties. So the field should be empty before the calculation is run. Is it not the case?

Reply:
Nope, it can have other properties, such as melting point etc.